Chatify/chat/consumers.py

```diff
@@ -8,26 +8,25 @@ async def connect(self):
         await self.accept()
 
     async def receive_json(self, event):
-        status = event.get("status")
-        json_data = {
-            "message": event,
-            "id": self.scope["user"].id,
-            "full_name": self.scope["user"].get_full_name(),
-            "profile_photo": self.scope["user"].profile_photo.url,
-            "status": status,
-        }
         await self.channel_layer.group_send(
             "visiblity-group",
             {
                 "type": "chat.message",
-                "text": json.dumps(json_data),
             },
         )
 
+    @sync_to_async
+    def updated_instance(self):
+        from .models import User
+
+        instance = User.objects.get(id=self.scope["user"].id)
+        return instance
+
```

Comment:
Here i can't get updated user is_online status that's why i have to write this function